src/entity/Events/CommonEventFactory.java

```diff
@@ -3,11 +3,12 @@
 import entity.Location.Location;
 import entity.Users.Organizer;
 
+import java.time.LocalDateTime;
 import java.util.ArrayList;
 
 public class CommonEventFactory implements EventFactory {
-    public Event create(Integer eventID, String eventName, Integer owner, Location location,
-                        ArrayList<Integer> peopleJoined, ArrayList<Integer> peopleWaitlisted, String time,
+    public Event create(Integer eventID, String eventName, String owner, Location location,
+                        ArrayList<String> peopleJoined, ArrayList<String> peopleWaitlisted, LocalDateTime time,
```

Comment:
It takes a lot of variables to instantiate an event-- in the future, we can refractor this part of the code to make this look cleaner.